app/graphql/types/category.rb

```diff
@@ -0,0 +1,34 @@
+class Types::Category < Types::BaseObject
+  description 'Information about a specific Category'
+
+  field :id, ID, null: false
+
+  field :title, String,
+    null: false,
+    description: 'The name of the category.'
+
+  field :description, String,
+    null: true,
+    description: 'A description of the category.'
```

Comment:
Can you change these to localized strings?  For now you can hardcode `{ en: title }` like we do in [Types::Media#synopsis](https://github.com/hummingbird-me/kitsu-server/blob/nuck/nolt-sso/app/graphql/types/media.rb#L33-L36) but it's likely that we'll want to localize them on the server instead of on the client